scripts/file_collector.py

```diff
@@ -0,0 +1,187 @@
+"""
+The purpose of this script is to collect all the /storage/ filepaths used by
+the p2a_impacts package.
+"""
+
+from argparse import ArgumentParser
+from sqlalchemy import create_engine
+from sqlalchemy.orm import sessionmaker
+import logging
+
+from ce.api.util import search_for_unique_ids
+from modelmeta import DataFile
+from p2a_impacts.parser import build_parse_tree
+from p2a_impacts.evaluator import evaluate_rule
+from p2a_impacts.fetch_data import (
+    get_dict_val,
+    read_csv,
+    translate_args,
+    get_models,
+)
+from p2a_impacts.utils import get_region, REGIONS
+
+
+def setup_logging(log_level):
+    formatter = logging.Formatter(
+        "%(asctime)s %(levelname)s: %(message)s", "%Y-%m-%d %H:%M:%S"
+    )
+    handler = logging.StreamHandler()
+    handler.setFormatter(formatter)
+    logger = logging.getLogger("scripts")
+    logger.addHandler(handler)
+    logger.setLevel(getattr(logging, log_level))
+    return logger
```

Comment:
This code exists in another part of the repo. Since we are doubling it up here we may want to consider moving it to the `utils` module and importing it in both places.